test/suites/console.sh

```diff
@@ -1,4 +1,14 @@
 test_console() {
+  lxc_version=$(lxc info | grep "driver_version: " | cut -d' ' -f4)
+  lxc_major=$(echo "${lxc_version}" | cut -d. -f1)
+
+  if [ "${lxc_major}" -lt 3 ]; then
+    echo "==> SKIP: The console ringbuffer require liblxc 3.0 or higher"
+    return
+  fi
```

Comment:
I skipped moving this into a separate function since it involves using `shift` and or returning variables and this check here is pretty brief.